cime_config/tests.py

```diff
@@ -140,8 +140,8 @@
     #atmopheric tests to mimic low res production runs
     "e3sm_atm_prod" : {
         "tests" : (
-            "SMS_Ln5.ne30pg2_r05_oECv3.F2010.eam-wcprod",
```

Comment:
Why not just modify the existing wcprod testmod dirs?

Reply:
@rljacob Sorry, I missed this comment. Presently, multiple tests that have different resolutions are sharing the same testmod dir. So, I had to create individual testmod dir for each test in which `user_nl_elm` has the appropriate `finidat`.